CHANGES.md

```diff
@@ -10,6 +10,8 @@
 
 <!-- Changes that affect Black's style -->
 
+- Format docstrings to have consistent quote placement (#2885)
```

Comment:
We should make it clear that this only affects the preview style. Probably we should have a separate section for the preview style.

Reply:
I opened #2890 to add a new CHANGELOG section for the preview style. (And also #2889 to format Black itself in preview mode, which also affects this PR.)

Reply:
I merged the previewstyle branch into this one, which puts this bullet point into the preview section. Hopefully that wasn’t an awful idea (I never know with Git branches!).